packages/create-app/index.js

```diff
@@ -155,7 +158,7 @@ async function init() {
         name: 'yes',
         initial: 'Y',
         message:
-          `Target directory ${targetDir} is not empty.\n` +
+          `Target directory ${baseDir} is not empty.\n` +
```

Comment:
I think that showing only the basename may be confusing, the user may think that a directory with that name could be created inside the current directory. I think that using current directory is more clear here:
```suggestion
          ( targetDir === '.' ? 'Current directory' : `Target directory ${targetDir} ) + ` is not empty.\n` +
```